### PR TITLE
add readme translation in german (close #3328)

### LINKED ISSUE #3328: Translation of README to German
It would be cool to have a German translation of the Hasura README.

## How to

- Create a new file called `README.german.md` inside the [/translations](https://github.com/hasura/graphql-engine/tree/master/translations) folder and add it to the table [here](https://github.com/hasura/graphql-engine/tree/master/translations#hasura-graphql-engine---translations)
- Translate the content
- Make sure all the links are working (including the table of content)
- Add your translation to the [English README](https://github.com/hasura/graphql-engine/blob/master/README.md#translations)

### PR DESCRIPTION
**Pull request by FynnGrandke**
### Description
Translated the English README to German. Let me know if there are any mistakes I have overseen!

I am open to any critics or other advices so this translation can make it to the docs! :)

If there is anything else - please feel free to tell me!

### Affected components
- [ ] Docs

If it is okay Issue #3328 can be closed!

If it is not okay, I'm happy to fix any issues!

Cheers,
Fynn

### CODE REVIEW

File: translations/README.german.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+* **Admin UI & Migration**: Admin UI & von Rails inspirierte Schemata-integratiomn
+* **Postgres** ❤️: Unterstuertzt Postgres Typen (PostGIS/geo-location, etc.), aendert Views zu *graphen*, loest gespeicherte Funktionen aus oder Prozesse mit Mutationen
+
+Mehr dazu auf [hasura.io](https://hasura.io) und in den [Docs](https://docs.hasura.io).
+
+## Inhaltsverzeichnis
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+**Table of Contents**
+
+- [Quickstart:](#quickstart)
```
Comment (marionschleifer) [inline translations/README.german.md]: Some links here are not working correctly.

Comment (FynnGrandke) [reply translations/README.german.md]: Thank you! Somehow I got that wrong but got it all working now!

**Comment by FynnGrandke** [comment]
Hey Ms. Schleifer!
I resolved the merging conflicts by pulling the master branch into it. Hope that was okay?!

Anything else I can do for regarding the ticket?

Cheers,
Fynn

**Comment by marionschleifer** [comment]
@FynnGrandke thank you for merging master 🙂 no, there is nothing else to do at the moment. I'm sure @shahidhk will review and merge this soon.

**Pull request merged**